config/initializers/routing_error_logging.rb

```diff
@@ -0,0 +1,16 @@
+module ActionDispatch
```

Comment:
One thing I suggest is to add a comment around the intent of this initializer, e.g., suppressing/overwriting logs for RoutingError exceptions by overwriting `log_error` method of `ActionDispatch:DebugExceptions`.